core/domain/html_cleaner.py

```diff
@@ -26,11 +26,12 @@
 from core.domain import rte_component_registry
 
 
-def filter_a(name, value):
-    """Returns whether the described attribute of an anchor ('a') tag should be
+def filter_a(unused_tag, name, value):
+    """Returns whether the described attribute of a tag should be
```

Comment:
Hi @seanlip, I'm not sure about this docstring

Reply:
Hi @NishealJ, thanks for flagging. I thought about this more and I actually have a different suggestion -- let's raise an exception (rather than return False) if the tag is not 'a'. The problems that would arise there would be in the code and we should alert developers immediately if they do it wrongly.

So, I would say: keep the original docstring, and add a part at the beginning that says:

    if tag != 'a':
        raise Exception('The filter_a method should only be used for 'a' tags.')

Have a test that catches that, too. Sorry for not realizing this earlier!

Reply:
Done Thanks